src/components/Records/Search/Input/SearchInput.vue

```diff
@@ -13,6 +13,17 @@
       <h2 class="d-none">
         Filter List
       </h2>
+
+      <!-- Search Box -->
+      <string-search
+        place-holder="Search through all data."
```

Comment:
placeholder is one word :)

Reply:
this is vue convention. when u use a capital letter in the names, it will chunk it down to dash-style when used as tags